From e0aaedea30483bf5b9bb085e12fff79144fa1174 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Thu, 29 Aug 2024 11:12:57 -0700
Subject: [PATCH] Disable portability warning as error

With latest automake 1.17 its warning about escape hash
mark [1] and since configure.ac uses -werror to call automake
this becomes an error and reconfigure fails.

escape hash mark is non-portable as discussed here [2]

Fow now let it be a warning, it should be fixed in a portable way

[1] https://debbugs.gnu.org/cgi/bugreport.cgi?bug=7610
[2] https://lists.gnu.org/archive/html/automake/2011-08/msg00023.html

Upstream-Status: Submitted [https://github.com/e2tools/e2tools/pull/31]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index dd32c4d..9f37591 100644
--- a/configure.ac
+++ b/configure.ac
@@ -14,6 +14,7 @@ AC_CONFIG_MACRO_DIR([m4])
 AM_INIT_AUTOMAKE([
 -Wall
 -Werror
+-Wno-portability
 1.9.6
 foreign
 subdir-objects
